jest.setup.js

```diff
@@ -0,0 +1,5 @@
+import { configureDefaults } from 'pleasantest';
+
+configureDefaults({
+  user: { targetSize: 35 /* px */ },
```

Comment:
Can you add a comment to describe why this default was set? Something like:
```suggestion
  // Design intent doesn't require Level AAA conformance, 
  // but we want to keep a baseline targetSize check
  user: { targetSize: 35 /* px */ },
```